Dockerfile.dbbrowser

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.21 AS builder
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY cmd/dbbrowser/ ./
+COPY pkg ./pkg
+RUN CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -o /dbbrowser
+
+FROM alpine:3
```

Comment:
this could probably be from Scratch, no ?

Reply:
I have to do `RUN apk add --no-cache ca-certificates`, I don't think scratch supports that, I'll double check though

Reply:
Yeah, it fails without alpine, and if you don't run that command you get an error when the apps starting
```
tls: failed to verify certificate: x509: certificate signed by unknown authority cloudrun
```